atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/api/cache/LockWatchValueScopingCacheImpl.java

```diff
@@ -211,7 +211,7 @@ private synchronized void updateStores(TransactionsLockWatchUpdate updateForTran
                 .keySet()
                 .forEach(timestamp -> cacheStore.createCache(StartTimestamp.of(timestamp)));
 
-        if (valueStore.getSnapshot().hasAnyTablesWatched()) {
+        if (valueStore.getSnapshot().hasAnyCellsWatched()) {
```

Comment:
this bit needs to know if _anything_ is watched, i.e. it needs to know if rows are watched but not tables.